client/src/utils/socket.js

```diff
@@ -6,21 +6,50 @@ import {
   addOnlineUser,
 } from "../store/conversations";
 
-const socket = io(window.location.origin);
+export const socketWrapper = {
```

Comment:
What is the difference between doing this and `export const socket = null` ?

Reply:
If I initialize it as `const` then I won't be able to assign it a value later on. In other words, I won't be able to initialize it. So I either have to do it like this `export let socket = null` or use the current approach.